source/js/hook-duoshuo.js

```diff
@@ -1,5 +1,5 @@
 if (typeof DUOSHUO !== 'undefined') hookDUOSHUO_tp();
-else $('#duoshuo-script')[0].onload = hookDUOSHUO_tp;
+else $('[src="http://static.duoshuo.com/embed.js"]')[0].onload = hookDUOSHUO_tp;

```

Comment:
Expected '{' and instead saw '$'.<br>Identifier 'hookDUOSHUO_tp' is not in camel case.